### DISCUSSION TITLE
[1913420]幻想曹操传2{本体}

### DISCUSSION THREAD

**Discussion by 605108259** [pjy612/SteamManifestCache#860] [Share]
### Steam 游戏ID

1913420

### 密钥列表

```hash
1913421;248fbb2b6d0615e765768d8d17fcdbba3307e14d81cafae28ab896d0f40b466d
```

### 清单附件

[1913421_474353582870998539.zip](https://github.com/user-attachments/files/17253879/1913421_474353582870998539.zip)

### 备注

605108259@qq.com

**Comment by pjy612** [comment]
* 密钥分隔符错误
* #26 
* 清单错误
* https://steamdb.info/app/1913420/depots/

**Reply by 605108259** [reply]
已更正

**Reply by pjy612** [reply]
......你这符号还是错的....
